### PR TITLE
Adds an Uninstall section to the LNBits bonus guide

### LINKED ISSUE #1114: [FEATURE REQ] Add a "Uninstall" section to the LNBits guide
### What & why

The[ LNBits bonus guide](https://raspibolt.org/guide/bonus/lightning/lnbits.html) needs a "Uninstall" section at the end as is now routinely done for all bonus guides.

### PR DESCRIPTION
**Pull request by VajraOfIndra**
#### What

This PR adds an Uninstall section to the LNBits bonus guide.

### Why

To remove LNBits and keep the node tidy and without unused files.

#### Scope

- [ ] significant change to core configuration
- [ ] independent bonus guide
- [x] minor change to bonus guide
- [ ] simple bug fix

Fixes #1114 

#### Test & maintenance

Uninstall LNBits from the node.

delete

### CODE REVIEW

File: guide/bonus/lightning/lnbits.md [hunk 1]
```diff
@@ -343,6 +344,61 @@ Updating to a [new release](https://github.com/lnbits/lnbits-legend/releases){:t
...
+
+* Stop and disable the systemd service and then delete the service file
+ 
+  ```sh
+  $ sudo systemctl disable lnbits.service
+  $ sudo systemctl stop lnbits.service
+  $ sudo rm /etc/systemd/system/lnbits.service
+  ```
+
+* Display the UFW firewall rules and notes the numbers of the rules for Thunderhub (e.g., X and Y below)
```
Comment (blckbx) [inline guide/bonus/lightning/lnbits.md:361]:
```suggestion
* Display the UFW firewall rules and notes the numbers of the rules for LNBits (e.g., X and Y below)
```

**Review by blckbx — COMMENTED** [review]
nit: Thunderhub -> LNBits

File: guide/bonus/lightning/lnbits.md (hunk 1 above)
Comment (VajraOfIndra) [reply guide/bonus/lightning/lnbits.md]: Thanks, commited your change 👍

**Review by blckbx — APPROVED** [review]
utACK

**Pull request merged**